<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="categoryLabel">
           <property name="text">
            <string>Job category</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="categoryLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="locationLabel">
           <property name="text">
            <string>Location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="locationLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="configLabel">
         <property name="text">
          <string>Path of CSV file of website data to scrape</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="configLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="configButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="stopwordLabel">
         <property name="text">
          <string>Path of stopword list (.txt format) (if not provided, will use default English stopword list)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="stopwordLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="stopwordButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="reportLabel">
         <property name="text">
          <string>Folder path for inserting reports</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="reportLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="reportButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="startScrapeButton">
           <property name="text">
            <string>Start scraping and keyword extraction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startWriteExcelButton">
           <property name="text">
            <string>Start generate Excel report</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="msgLabel">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
